app/Http/Controllers/IncomingTextController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\TwilioRequest as Request;
+
+class IncomingTextController extends Controller
+{
+    protected $commands = [
+        "unsubscribe" => [
+            "STOP",
+            "STOPAll",
+            "UNSUBSCRIBE",
+            "CANCEL",
+            "END",
+            "QUIT",
+        ],
+        "subscribe" => [
+            "START",
+            "YES",
+        ]
+    ];
+
+    /**
+     * Starting point for all incoming text messages.
+     */
+    public function receiveText(Request $request)
+    {
+        if ($command = $this->translateCommand($request->Body)) {
+            return $this->{$command}($request);
+        }
+
+        return $this->responseMessage('Unrecognized command: '.$request->Body);
+    }
+
+    protected function subscribe(Request $request)
+    {
+        $request->phoneNumber()->markBlacklisted(false);
```

Comment:
Any reason we couldnt' instead use `addToBlackList` and `removeFromBlacklist`? I know it's not quite as concise, but it means we don't have to worry about boolean parameters.